### PR TITLE
Update the display of gcc -v

### PR DESCRIPTION
**Pull request by er-1**
`gcc -v` will print the FSF revision number the IBM branch in based upon, and the IBM branch revision number.

Signed-off-by: Erwan Prioul <erwan@linux.ibm.com>

### CODE REVIEW

File: configs/11.0/packages/gcc/stage_1 [hunk 1]
```diff
@@ -31,8 +31,8 @@ atcfg_pre_hacks() {
 	echo "Advance-Toolchain-${at_major_internal}" > ${ATSRC_PACKAGE_WORK}/gcc/DEV-PHASE
 	# Overwrite the existing REVISION with the current SVN revision date
 	mv ${ATSRC_PACKAGE_WORK}/gcc/REVISION ${ATSRC_PACKAGE_WORK}/gcc/REVISION.back
-	sed "s/branch merged/branch, revision: ${ATSRC_PACKAGE_REV} merged/" ${ATSRC_PACKAGE_WORK}/gcc/REVISION.back \
-		 > ${ATSRC_PACKAGE_WORK}/gcc/REVISION
+	sed "s/ \([^ ][^ ]*\)]$/ ${ATSRC_PACKAGE_REV}, based upon FSF revision \1]/" \
+	        ${ATSRC_PACKAGE_WORK}/gcc/REVISION.back > ${ATSRC_PACKAGE_WORK}/gcc/REVISION
```
Comment (tuliom) [inline configs/11.0/packages/gcc/stage_1]: I think this code needs a mechanism to detect when sed does not match anything.  This way we would catch this kind if issues earlier.
Maybe a diff between REVISION.back and REVISION?

Comment (tuliom) [reply configs/11.0/packages/gcc/stage_1]: Does AT next also come with a gcc/REVISION or do we need to generate it?

The older AT versions (at least 9.0 and 10.0) also need this fix.

**Comment by er-1** [comment]
@tuliom as asked I've restored the symlink between next and AT 11.0.

**Comment by er-1** [comment]
Fixes #497

**Pull request merged**